.github/workflows/publish.yml

```diff
@@ -29,13 +29,41 @@ jobs:
         run: ./gradlew qualifyVersionGha
       - name: run checks
         id: checks
-        run: ./gradlew check
+        run: ./gradlew check -Pjunit-tags=!slow -x jcstress
+
+  racesChecks:
+    # similar limitations as in prepare, but we parallelize the slowest tests here (races)
+    name: racesChecks
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: run racing tests
+        id: racesTests
+        run: ./gradlew test -Pjunit-tags=races
+
```

Comment:
```suggestion
```

seems to be no race tests

Reply:
weird? the whole point is that a bunch of raceSomething tests take up the majority of the time (9min) on CI, so while it is difficult to split them further, it makes sense to have them execute separately. if anything, loop + jcstress + prepare would take about 10min together so that could be the axis to split on.

Reply:
oh. My bad. it has tag `race` rather than `races` that was the reason why I have not seen anything

Reply:
should be fixed to `race` than

Reply:
damn, yeah I changed it last minute in the yaml but not in the actual tags 🤦